src/main/webapp/mapScript.js

```diff
@@ -2,6 +2,33 @@
 function initMap() {
   const map = new google.maps.Map(document.getElementById("map"), {
     center: { lat: -34.937, lng: 150.644 },
-    zoom: 8
+    zoom: 14
   })
+
+  if (navigator.geolocation) {
+    navigator.geolocation.getCurrentPosition(function(position) {
```

Comment:
Maybe just add a couple comments to clarify what exactly you are doing. For example, at first I didn't understand what "if (navigator.geolocation)" was checking until below I saw the comment for the else statement.

Reply:
And not specifically this line, but in general.

Reply:
+1

Reply:
Okay, will do. Thanks